package.json

```diff
@@ -26,20 +33,29 @@
     "typescript": "^3.5.0"
   },
   "dependencies": {
```

Comment:
`express` and `passport` should probably be moved into `peerDependencies`

Reply:
fixed in 4a93ba7